### DISCUSSION TITLE
AC Microcourses Feedback - Hello World

### DISCUSSION THREAD

**Discussion by sgbaird** [AccelerationConsortium/ac-microcourses#11] [Hello World]
Use this discussion to provide feedback about the Hello World microcourse (what to keep, what to change, what to remove) and other comments or feedback you'd like to share.

**Comment by evelynnesher** [comment]
I wanted to share some thoughts about the microcourses and get feedback from others on how they're finding the content so far. I’ve been moving through the material and, while I’m understanding the concepts, I’m finding the coding portions to be a bit challenging. There seems to be a lot introduced at once, and even with my coding experience, I’ve been struggling more than I expected.

It seems like the assignment at the end of each module is the only significant coding task, but I think having at least one more coding component earlier on would be really helpful for reinforcing the skills. One suggestion I have is to introduce optional “checkpoints,” “hints,” or “feeling stuck?” sections throughout the course. These could provide code snippets to assist with the current tasks while offering reusable tools and functions for future projects. Another option could be adding mini quizzes focused on coding to reinforce understanding.

I’d love to hear if others are having a similar experience or have additional thoughts!

**Comment by sgbaird** [comment]
Copying following feedback from SS:

> Thanks for creating such a great course. I really enjoyed it.
>
> On the UofT website, the approximate duration was 25 hours, but I think if someone wants to watch the YouTube videos and read the other resources, it takes more. In my case, it was more than 50 hours (totally worth it).
>
> To better connect the course to materials discovery, I also think the “Materials Informatics” playlist on Taylor Sparks’ YouTube channel is amazing. I watched the ones included in the course, and I can’t wait to find time to watch the full playlist.
>
> Thanks again, and I hope more people can use this course.
